tests/factories/transcript.py

```diff
@@ -10,20 +10,6 @@ class Meta:
 
     video_id = Sequence(lambda n: f"video_id_{n}")
     transcript_id = Sequence(lambda n: f"transcript_id_{n}")
-    transcript = [
-        {
-            "text": "[Music]",
-            "start": 0.0,
-            "duration": 7.52,
-        },
-        {
-            "text": "how many of you remember the first time",
-            "start": 5.6,
-            "duration": 4.72,
-        },
-        {
-            "text": "you saw a playstation 1 game if you were",
-            "start": 7.52,
-            "duration": 4.72,
-        },
-    ]
+    transcript = Sequence(
+        lambda n: [{"text": "[Music]", "start": float(n), "duration": float(n + 1)}]
+    )
```

Comment:
As we only get the transcript text value back, not the transcript object, having different ones allows us to compare them and tell the difference.

Reply:
:+1:

This always generates lists of only one dict, do we want to make it generate a few or maybe a random number of dicts? Could also replace `"[Music]"` with lorem text from `faker`